.changeset/weak-kangaroos-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/graph': patch
+---
+
+Fix warnings and TypeScript errors for tests in xstate/graph
```

Comment:
This PR only touches test files so it doesn't need a public-facing changeset file
```suggestion
---
---
```